.gitignore

```diff
@@ -10,8 +10,8 @@ awsconfig
 /public_gen
 /public/vendor/npm
 /tmp
-vendor/phantomjs/phantomjs
-vendor/phantomjs/phantomjs.exe
+contrib/phantomjs/phantomjs
+contrib/phantomjs/phantomjs.exe
```

Comment:
maybe tools is better than contrib? another option would be to place this under bin dir that is already git ignored

Reply:
using `bin` feels weird from a distribution standpoint. `grafana-server` endsup in `/usr/sbin` while phantomjs endsup in `/usr/share/grafana/phantomjs`

`tools` sounds better than `contrib`. Phantomjs is something grafana uses, not something that has been contributed to the project.

Changing to `tools`